### PR TITLE
Xray processor

### LINKED ISSUE #60: xray_image_classification_processor
this issue is created to write a processor for x-ray image classification.

we will be using the HuggingFace based VIT pretrained model https://huggingface.co/nickmuchi/vit-finetuned-chest-xray-pneumonia in our task.

we have verified the accuracy claim of 95.51 % for this model on https://data.mendeley.com/datasets/jctsfj2sfn/1 

this processor and reader from issue #59 will help to create an application of x-ray image classification using the ForteHealth pipeline.

### PR DESCRIPTION
**Pull request by bhaskar2443053**
This PR fixes #60 

### Description of changes
this pr will add a new processor to ForteHealth to perform x-ray image classification.

### Possible influences of this PR.
this implementation needs torch and transformer libs installed

### Test Conducted
Describe what test cases are included for the PR.

### CODE REVIEW

**Comment by Piyush13y** [comment]
@bhaskar2443053 You have 2 PRs in place. And I see that file `xray_image_reader.py` is being created in both the PRs. Do you wish to abandon PR #61 since its redundant or am I missing something?

**Comment by bhaskar2443053** [comment]
> @bhaskar2443053 You have 2 PRs in place. And I see that file `xray_image_reader.py` is being created in both the PRs. Do you wish to abandon PR #61 since its redundant or am I missing something?

oh its due to the #62 is created from the branch of pr #61.
let me close the pr #62 and recreate from master branch.

as i think we show have independent pr for reader and processor.
